src/utils/backupUser.json

```diff
@@ -0,0 +1,169 @@
+{
```

Comment:
The `sampleUsers.json` looks good enough and if this is not needed, please feel to delete this one.

Reply:
Oh shit that was a personal file, I was using ! I will make sure to delete it